java/component/keyple-calypso/src/main/java/org/eclipse/keyple/calypso/transaction/PoTransaction.java

```diff
@@ -71,11 +71,12 @@
      * to the length of the outgoing data plus 6 bytes
      */
     private static final int SESSION_BUFFER_CMD_ADDITIONAL_COST = 6;
+    private static final int APDU_HEADER_LENGTH = 5;
 
     /** Ratification command APDU for rev <= 2.4 */
-    private static final byte[] RATIFICATION_CMD_APDU_LEGACY = ByteArrayUtil.fromHex("94B2000000");
+    private static final byte[] RATIFICATION_CMD_APDU_LEGACY = ByteArrayUtil.fromHex("94B2010000");
```

Comment:
We may extract RATIFICATION_CMD_APDU_LEGACY & RATIFICATION_CMD_APDU and define them in a public interface.
And in PoSecuritySettings, we could maybe propose a way to personalize the ratification cmd.

Reply:
The Calypso requirements define 00B2010000 as ratification command. Do we need to propose a way to define something different?

Reply:
Anyway, we need to improve the ratification command building.